tests/test_slim_engine.py

```diff
@@ -1003,7 +1003,7 @@ def verify_mutation_rates(self, contig, ts):
             self.verify_rates_match(mut_rate, intervals, slim_rates, slim_ends)
 
         # also verify any bits not covered by DFEs have no mutations
-        empty_intervals = np.empty((0, 2))
+        empty_intervals = np.array([[0, int(contig.length)]])
```

Comment:
This is not an empty interval any more? We're needing to check that anything *not covered* by an interval works right, and if we start with these intervals we won't have any empty intervals.